eng/performance/benchmark_jobs.yml

```diff
@@ -90,16 +90,19 @@ jobs:
         container: ${{ parameters.container }}
         strategy:
           matrix:
-            ${{ each framework in parameters.frameworks }}:
-              ${{ framework }}:
-                _Framework: ${{ framework }}
+            ${{ each channel in parameters.channels }}:
+              ${{ channel }}:
+                _Channel: ${{ channel }}
                 _Configs: CompilationMode=Tiered RunKind="${{ parameters.kind }}"
                 _PerfLabArguments: $(PerfLabArguments)
-                _BuildConfig: ${{ parameters.architecture }}_$(_Framework)_${{ parameters.kind }} # needs to be unique to avoid logs overwriting in mc.dot.net
+                _BuildConfig: ${{ parameters.architecture }}_$(_Channel)_${{ parameters.kind }} # needs to be unique to avoid logs overwriting in mc.dot.net
```

Comment:
this is changing CI build ID used in the Build Status Page in the README.md file. It would be nice to update the Build Status Page or just remove it and keep the two most important buttons (Public Build Status and Internal Build Status)

Reply:
Thanks! I wasn't aware of this dependency. Will update this.